### PR TITLE
add gazebo tag in vacuum_gripper.xacro

### PR DESCRIPTION
**Pull request by knorth55**
set `gazebo` tag in vacuum gripper

### CODE REVIEW

File: jsk_2016_01_baxter_apc/robots/left_vacuum_gripper.xacro [hunk 1]
```diff
@@ -1,6 +1,22 @@
...
+      <selfCollide>true</selfCollide>
+    </gazebo>
+    <gazebo reference="left_gripper_vacuum_pad_base">
+      <selfCollide>true</selfCollide>
+    </gazebo>
+    <gazebo reference="left_gripper_vacuum_pad">
+      <selfCollide>true</selfCollide>
+    </gazebo>
+    <gazebo reference="left_gripper_vacuum_pad_joint">
+      <implicitSpringDamper>1</implicitSpringDamper>
```
Comment (pazeshun) [inline jsk_2016_01_baxter_apc/robots/left_vacuum_gripper.xacro]: This is bool type, isn't it?

Comment (knorth55) [inline jsk_2016_01_baxter_apc/robots/left_vacuum_gripper.xacro]: Your review is correct, thank you.
in that case, bater_description is wrong...

Comment (pazeshun) [inline jsk_2016_01_baxter_apc/robots/left_vacuum_gripper.xacro]: www

File: jsk_2016_01_baxter_apc/robots/left_vacuum_gripper.xacro [hunk 2]
```diff
@@ -1,6 +1,22 @@
 <?xml version="1.0"?>
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="left_vaccum_gripper">
   <xacro:include filename="$(find jsk_2016_01_baxter_apc)/robots/common_constants.xacro"/>
+  <xacro:arg name="gazebo" default="false"/>
+  <xacro:if value="$(arg gazebo)">
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/robots/left_vacuum_gripper.xacro]: Why this condition is required?

File: jsk_2016_01_baxter_apc/robots/right_vacuum_gripper.xacro [hunk 3]
```diff
@@ -1,6 +1,22 @@
 <?xml version="1.0"?>
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="right_vaccum_gripper">
   <xacro:include filename="$(find jsk_2016_01_baxter_apc)/robots/common_constants.xacro"/>
+  <xacro:arg name="gazebo" default="false"/>
+  <xacro:if value="$(arg gazebo)">
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/robots/right_vacuum_gripper.xacro]: Why is this condition required?

File: jsk_2016_01_baxter_apc/robots/left_vacuum_gripper.xacro (hunk 2 above)
Comment (knorth55) [inline jsk_2016_01_baxter_apc/robots/left_vacuum_gripper.xacro]: gazebo tag is only for simulation

Comment (wkentaro) [inline jsk_2016_01_baxter_apc/robots/left_vacuum_gripper.xacro]: Is not ignored?

Comment (knorth55) [inline jsk_2016_01_baxter_apc/robots/left_vacuum_gripper.xacro]: I've never tried, i will try it

Comment (wkentaro) [inline jsk_2016_01_baxter_apc/robots/left_vacuum_gripper.xacro]: Thanks.

Comment (knorth55) [inline jsk_2016_01_baxter_apc/robots/left_vacuum_gripper.xacro]: i tested and it does not matter

**Comment by pazeshun** [comment]
Ping @wkentaro

**Comment by knorth55** [comment]
please merge this

**Pull request merged**